### PR TITLE
Install provisioning profile

### LINKED ISSUE #3: Support for provisioning profile file path
We could use a new environment variable `TAB_PROVISIONING_PROFILE_FILEPATH` to define the path of profile to be installed

### PR DESCRIPTION
**Pull request by lucianomarisi**
Added new configuration to install a provisioning profile to work with Xcode 8.

`TAB_PROVISIONING_PROFILE_PATH` is now used to specify the path of the  provisioning profile to install

This may be better as a fastlane action, so if anybody with decent ruby skills could help that would be much appreciated

fixes #3

### CODE REVIEW

**Comment by Neil3079** [comment]
I think we should hold off on this until we can use it as an action and let people opt in to it, I'm fairly sure if a user has their own before all hook this won't be called (I could be wrong here though)

**Comment by lucianomarisi** [comment]
@Neil3079 the opt out would be about defining the environment variable `TAB_PROVISIONING_PROFILE_PATH ` or not. the `before_all` question I'll have a check

**Comment by lucianomarisi** [comment]
@Neil3079 the opt out would be about defining the environment variable `TAB_PROVISIONING_PROFILE_PATH` or not. 

Just checked and the `before_all` is a problem since it can be overridden. If this was an action or function  (like the existing `setup`) one, would you call it from all the place `gym` is used?

**Comment by Neil3079** [comment]
I would suggest action for this personally and yes, but i would leave it to the end user to specify that it needs to happen in their `Fastfile`. This is just a personal preference however and I am open to discussion

**Comment by lucianomarisi** [comment]
I agree with making it an action instead of function, I just haven't been able to figure out how to do it.

Not sure about getting the user manual call this action. I'd prefer to specify the `TAB_PROVISIONING_PROFILE_PATH` and use the basic MasterFastfile.

**Comment by Neil3079** [comment]
Ok, the action > function is what I'm most fussed about on this one, so lets leave this branch  (as you can still use it in the meantime) until we figure out how to make it an action (Id suggest that the two of us take a look on friday).

I see your point about needing to do less things to opt in, so would be ok with us dropping making the end user specify in their before_all and doing it implicitly with gym actions

**Comment by lucianomarisi** [comment]
@Neil3079 sounds good, let's have a look on friday on the action, the meantime the branch is fine

**Review by Neil3079 — APPROVED** [review]

**Pull request merged**
